lib/models/liveValidationError.js

```diff
@@ -21,7 +21,7 @@
  */
 class LiveValidationError {
   constructor(code, message) {
-    this.code = code;
+    this.code = code.name;
```

Comment:
ooppss I wouldn't change this. As this is auto generated from Autorest from swagger.